en/airframes_multicopter/qav-r-5-kiss-esc-racer.md

```diff
@@ -0,0 +1,186 @@
+# QAV-R 5" KISS ESC Racer
+
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/preview.jpg)
+![](../../assets/airframes/multicopter/qav-r-5-kiss-esc-racer/preview2.jpg)
+
+## Parts List
+
+### Vehicle (needed for flying)
+* Autopilot: [Pixracer](../flight_controller/pixracer.md) from AUAV including power- and WiFi-module
```

Comment:
Which power module? I know you mention it specifically later, but I'd name it here too.